crates/sui-core/src/unit_tests/batch_transaction_tests.rs

```diff
@@ -57,15 +57,16 @@ async fn test_batch_transaction_ok() -> anyhow::Result<()> {
             )
             .unwrap();
     }
-    let data = TransactionData::new_programmable_with_dummy_gas_price(
+    let data = TransactionData::new_programmable(
         sender,
         vec![authority_state
             .get_object(&all_ids[N])
             .await?
             .unwrap()
             .compute_object_reference()],
         builder.finish(),
-        MAX_GAS_BUDGET_FOR_TESTING,
+        rgp * TEST_ONLY_GAS_UNIT_FOR_OBJECT_BASICS * 10,
```

Comment:
comment why x 10?

Reply:
`rgp * TEST_ONLY_GAS_UNIT_FOR_OBJECT_BASICS` is too low. Some tests need special casing